src/NosCore.GameObject/Quest.cs

```diff
@@ -54,10 +55,30 @@ public QstiPacket GenerateQstiPacket(bool showDialog)
                 QuestObjectiveSubPackets = objectives
             };
         }
+
+
     }
 
     public class Quest : QuestDto
     {
+        public TargetPacket GenerateTargetPacket()
+        {
+            return new TargetPacket
+            {
+                QuestId = QuestId, TargetMap = TargetMap ?? 0, TargetX = TargetX ?? 0,
+                TargetY = TargetY ?? 0
+            };
+        }
+
+        public TargetOffPacket GenerateTargetOffPacket()
+        {
+            return new TargetOffPacket
+            {
+                QuestId = QuestId, TargetMap = TargetMap ?? 0, TargetX = TargetX ?? 0,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the '??=' operator here.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5304413)